test/test_trixi2vtk.jl

```diff
@@ -5,6 +5,21 @@ using Trixi2Vtk
 using ReadVTK
 
 
+# Get the version of Trixi.jl we are testing since the output file name
+# format changed in v0.8.0
+import Pkg
+using UUIDs: UUID
+const LEADING_ZEROS = let
```

Comment:
```suggestion
const LEADING_ZEROS_TRIXI_v8 = let
```
Should we make this more descriptive (honest question)?

Reply:
I would find this a bit confusing since 
- it also contains leading zeros on other versions (zero of them)
- it will likely have the same number of leading zeros in Trixi.jl v0.9

Reply:
OK, then let's keep it.